bmorph/core/workflows.py

```diff
@@ -480,11 +529,11 @@ def unpack_and_write_netcdf(results, segs, file_path, out_varname='scbc_flow', m
             config_dir=bmorph_config['data_path']+'/mizuroute_configs/',
             topo_dir=bmorph_config['data_path']+'/topologies/',
             input_dir=bmorph_config['data_path']+'/input/',
-            output_dir=bmorph_config['data_path']+'/output/',
+            output_dir=bmorph_config['data_path']+bmorph_config['output_sub'],
```

Comment:
Could you change this to `bmorph_config.get('output_dir', '/output/')` so that it defaults to `/output/` like before? Also, I like `output_dir` rather than `output_sub`, as it's a bit easier to understand.

Reply:
I fix this in PR90 to be just `output_path` ... so this change can either be dropped or quickly patched

Reply:
I'd still like it to be done using `.get` so that we have a sensible default that doesn't break backwards compatibility.

Reply:
oh there is a default in PR 90 .... let me nab how i handle it

Reply:
`# need to set our optional directory configs to their default
    # in case they were not specified
    for optional_config, default in zip(['mizuroute_configs_path', 'topologies_path', 'input_path', 'output_path'], ['/mizuroute_configs/', '/topologies/', '/input/', '/output/']):
        if optional_config not in bmorph_config.keys():
            bmorph_config[optional_config] = bmorph_config['data_path'] + default
    if not 'out_name' in bmorph_config.keys():
        bmorph_config['out_name'] = None

    config_path, mizuroute_config = mizutil.write_mizuroute_config(
            bmorph_config["output_prefix"],
            scbc_type, bmorph_config['apply_window'],
            config_dir=bmorph_config['mizuroute_configs_path'],
            topo_dir=bmorph_config['topologies_path'],
            input_dir=bmorph_config['input_path'],
            output_dir=bmorph_config['output_path'],
            out_name=bmorph_config['out_name']
            )`

Reply:
lines 527-543 in PR90's workflows.py ... this is still backwards compatible as it sets it to what it was before

Reply:
I'll get rid of this modification for this PR to keep it about the hruless segs